tests/views_test.py

```diff
@@ -1,5 +1,100 @@
-from src_api.views import helloworld
+from numpy import ndarray, result_type
+from src_api.views import *
+import shutil
+import os
 
+def TestBasics():
```

Comment:
Capital letters are usually used for classes.
Please used best practice code formatting rules for functions (small caps)